.travis.yml

```diff
@@ -17,8 +17,8 @@ env:
     - CCACHE_MAXSIZE=500Mi
     - CCACHE_CPP2=1
     - MAINTAINER_MODE=ON
-    - WITHOUT_TOKUDB=ON
-    - WITHOUT_ROCKSDB=ON
+    - WITHOUT_TOKUDB=OFF
```

Comment:
Maybe inverted WITH_TOKUDB=ON WITH_ROCKSDB=ON would be more readable?

Reply:
I will change it in the future because it should be done staring from 5.6.